Program/Sound

```diff
@@ -0,0 +1,121 @@
+import random
+
+frq = 12500
+amp = 10
+chan = 1
+hz = 22050
+numsamples = 10
+S = sineWave(600, 2000, 5)
+D = sineWave(440, 2000, 5)
+  
+def sineWave(freq, amplitude, length): 
+  # Create a blank sound 
+  buildSin = makeEmptySoundBySeconds(length)
+  # Set constants 
+  samplingRate = getSamplingRate(buildSin) 
+  interval = 1.0 / freq 
+  samplesPerCycle = interval * samplingRate 
+  maxCycle = 2 * pi 
+  # Generate the sound 
+  for pos in range(getLength(buildSin)): 
+    rawSample = sin((pos / samplesPerCycle) * maxCycle) 
+    sampleVal = int(amplitude * rawSample) 
+    setSampleValueAt(buildSin, pos, sampleVal) 
+  return buildSin
+  
+def mergeSine(sound1, sound2):
```

Comment:
Is this the best function name? Could this be redefined using a list rather than two variables?